ui/lib/core/addon/components/info-table-item-array.hbs

```diff
@@ -1,51 +1,72 @@
-<div data-test-info-table-item-array {{did-insert (perform this.fetchOptions)}}>
+{{! the class linkable-item is needed for the read-more component }}
+<div data-test-info-table-item-array {{did-insert (perform this.fetchOptions)}} class="linkable-item">
   {{#if @isLink}}
     <div data-test-row-value={{@label}}>
-      {{#each this.displayArrayAmended as |name|}}
-        {{#if (is-wildcard-string name)}}
-          {{#let (filter-wildcard name this.allOptions) as |wildcardCount|}}
-            <span>{{name}}</span>
-            <span class="tag is-light has-text-grey-dark" data-test-count="{{if wildcardCount wildcardCount 0}}">
-              includes
-              {{if wildcardCount wildcardCount 0}}
-              {{if (eq wildcardCount 1) @wildcardLabel (pluralize @wildcardLabel)}}
-            </span>
-            {{#if (eq this.displayArrayAmended.lastObject name)}}
-              <LinkTo @route={{this.rootRoute}} @query={{hash tab=@queryParam}}>
-                <span data-test-view-all={{@viewAll}}>View all {{@viewAll}}</span>
+      <ReadMore>
```

Comment:
Tested this for all other use cases and it actually adds a nice ux. It improves the experience all around.

Reply:
Good question. But yep. DispalyArrayAmended deals with issues where doNotTruncate is false (in transform and database) and we want to amend the array down from >= 10 to 5 and then link to another tab that filters for only those array items.